transcom/mymove
# MB-16587: update move submission email

## [Jira ticket](https://dp3.atlassian.net/browse/MB-16587)

## Summary

This updates the email notification that is sent out when a move is submitted. 

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. To make the email sending works locally follow the steps here: https://dp3.atlassian.net/wiki/spaces/MT/pages/1467973894/How+to+test+sending+email+locally
2. Create a new user and make sure to use your email address as the contact info.
3. Create and submit an HHG shipment.

## Screenshots

![Screenshot 2023-08-10 at 2 37 00 PM](https://github.com/transcom/mymove/assets/84742904/7c7aa7b2-39c4-4130-8a2e-862a8b5ff007)

![Screenshot 2023-08-10 at 2 55 46 PM](https://github.com/transcom/mymove/assets/84742904/8aa2cc6b-3c98-4b63-8477-b21a2f164ef0)

![Screenshot 2023-08-10 at 2 45 55 PM](https://github.com/transcom/mymove/assets/84742904/61261363-4b15-4373-ae77-effdaacd5653)
